hs_core/search_indexes.py

```diff
@@ -531,6 +532,15 @@ def prepare_resource_type(self, obj):
         """Resource type is verbose_name attribute of obj argument."""
         return obj.verbose_name
 
+    def prepare_content_type(self, obj):
+        if obj.verbose_name != 'Composite Resource':
+            return [obj.discovery_content_type]
+        else:
+            output = []
+            for f in obj.logical_files:
+                output.append(f.get_discovery_content_type())
```

Comment:
@alvacouch  Not sure if would be a problem for discovery if the ```output``` list contains duplicate content types.

Reply:
It's not really a problem. SOLR mitigates.

Reply:
Sounds good.